config/rbac/role.yaml

```diff
@@ -101,6 +101,15 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
Why do we add new RBAC on removal of a feature?

Reply:
It's just reshuffling of existing permissions. I will look into it.

Reply:
done